manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Mixxx.Mixxx
+PackageVersion: 2.5.0
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  InstallLocation: INSTALL_ROOT="<INSTALLPATH>"
+UpgradeBehavior: install
+ReleaseDate: 2024-12-21
+AppsAndFeaturesEntries:
+- ProductCode: '{660C0DCB-681F-4843-B210-293523BB8BD7}'
+  UpgradeCode: '{921DC99C-4DCF-478D-B950-50685CB9E6BE}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi
```

Comment:
This is a beta version @Dragon1573

Reply:
But [it says](https://mixxx.org/download/#stable) itself a stable version ... 😕

![image](https://github.com/user-attachments/assets/19481aa7-64dd-4369-a983-2ce799cf367e)

Reply:
> [!WARNING]
> 
> Due to signing issues, we can't provide a signed `2.5.0` release now. In the meantime please use `2.5-beta-152` which uses already the final source code of _Mixxx_ `2.5.0`.
>
> _Originally posted at [Mixxx - Download the Best Free DJ Mixing Software App](https://mixxx.org/download/#stable)_

![image](https://github.com/user-attachments/assets/efd6e749-67f3-4a50-969c-3826b2f7507f)

Reply:
OK...I was wondering why my bot didn't work. 🤔